lib/tasks/ont_runs.rake

```diff
@@ -83,3 +83,37 @@ def create_libraries(count:)
 
   puts '-> Successfully created ONT libraries'
 end
+
+def submit_create_run_query(variables:, library_names:)
+  flowcells = variables.flowcells(library_names: library_names)
```

Comment:
Instead of passing the `variables` object in here then just calling one method on it, would it be tidier to call this method in `create_runs` and pass in the `flowcells` instead of `variables`?

Reply:
Try telling that to rubucop 🙄. I'll give it another go

Reply:
I'm choosing not to fix this as rubocop is rejecting any of my attempts